src/index.js

```diff
@@ -24,34 +24,44 @@ var KNOWN_STATICS = {
     arity: true
 };
 
-var defineProperty = Object.defineProperty;
-var getOwnPropertyNames = Object.getOwnPropertyNames;
-var getOwnPropertySymbols = Object.getOwnPropertySymbols;
-var getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
-var getPrototypeOf = Object.getPrototypeOf;
-var objectPrototype = getPrototypeOf && getPrototypeOf(Object);
+const defineProperty = Object.defineProperty;
+const getOwnPropertyNames = Object.getOwnPropertyNames;
+const getOwnPropertySymbols = Object.getOwnPropertySymbols;
+const getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
+const getPrototypeOf = Object.getPrototypeOf;
```

Comment:
If your commit has the chance to be merged it would be great if you can also remove lines 27 to 30 and replace them by:
```javascript
const {defineProperty, getOwnPropertyNames, getOwnPropertySymbols, getOwnPropertyDescriptor} = Object
```

Reply:
I had that initially, but the project targets node 4 too - which means object dereferencing isn't supported.

Reply:
Oh, I see, I wonder why such requirement but in any case at least you get rid of those annoying vars :slightly_smiling_face: